### PR TITLE
add utils/devtools for plain objects

### LINKED ISSUE #50: Is there any devtools yet?
Just like what the title said.

### PR DESCRIPTION
**Pull request by dai-shi**
close #50

### CODE REVIEW

**Comment by dai-shi** [comment]
@rohmanhm 
https://codesandbox.io/s/react-typescript-forked-62616
Is this something you'd expect?

**Comment by rohmanhm** [comment]
@dai-shi Definitely 💯 
Thanks for the quick update. You're awesome.

File: src/utils.ts [hunk 1]
```diff
@@ -47,3 +47,70 @@ export const subscribeKey = <T extends object>(
...
+  try {
+    extension = (window as any).__REDUX_DEVTOOLS_EXTENSION__
+  } catch {}
+  if (!extension) {
+    if (
+      typeof process === 'object' &&
+      process.env.NODE_ENV === 'development' &&
+      typeof window !== 'undefined'
+    ) {
+      console.warn('Please install/enable Redux devtools extension')
```
Comment (Aslemammad) [inline src/utils.ts:73]: For better messages, Can we use `[Warning] Please....` here?

**Review by Aslemammad — CHANGES_REQUESTED** [review]

File: src/utils.ts (hunk 1 above)
Comment (dai-shi) [reply src/utils.ts:73]: I don't have strong opinions on this message. Thought it's redundant, but if you think it's better, I can add it for sure.

**Comment by dai-shi** [comment]
Does anyone want to play with it before merging?

**Review by Aslemammad — COMMENTED** [review]
> Does anyone want to play with it before merging?

I tested it with codesandbox and Redux devtools, it works perfectly as I see.

**Comment by dai-shi** [comment]
Published: https://www.npmjs.com/package/valtio/v/0.4.9

**Pull request merged**